Export_gen_files/config.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+#title           :config.py
+#description     :Config file
+#author          :Diego Alvarez
+#date            :2016-06-05
+#python_version  :3.5  
+#==============================================================================
+
+##############################
+#Replace with your own values
+##############################
+
+#Default values to be used in gentotsv.py
+source_dir_oxford = "/media/sf_E_DRIVE/PFE/TestMigrationOxford/Data_sample_15/"
```

Comment:
Is there a way to use relative paths, so that we don't have to change the config in say 80% of scenarios ?